app/mailers/notify_mailer.rb

```diff
@@ -27,6 +27,12 @@ def application_returned_email(application_id)
     mail(to: provider_email)
   end
 
+  def access_granted_email(email)
```

Comment:
Its kind of interesting in a way now we have two ways to create emails - I kind of like the subscription based side-effect method. I know we are not creating events for use creation, but it is a nice pattern.

Perhaps there is a refactor in there at some point where we use subscribers for these kinds of things...

Not for this PR though :)

Reply:
Yes, I'd thought the same. I suspect that may be required when we add the assurance logging for user management.